AutomationRebuild/src/test/java/JDBC/JDBCTest.java

```diff
@@ -0,0 +1,322 @@
+package JDBC;
+
+import org.testng.annotations.Test;
+import static org.testng.Assert.assertEquals;
+
+import java.sql.*;
+
+public class JDBCTest {
+	@Test
+	public void problem1() throws SQLException {
+		final String QUERY = "SELECT city FROM sakila.city ORDER BY city DESC LIMIT 10";
+		String[] cities = new String[10];
+		int index = 0;
+		Connection connection = DriverManager
+				.getConnection("jdbc:mysql://localhost:3306/sakila?useSSL=false", "root", "password");
+
+		try {
+			Statement stmt = connection.createStatement();
+			ResultSet rs = stmt.executeQuery(QUERY);
+			while (rs.next()) {
+				cities[index++] = rs.getString("city");
+			}
+		} catch (SQLException e) {
+			System.out.println("problem 1 error");
+			e.printStackTrace();
+		}
+		finally {
+			if(connection != null) {
+				connection.close();
+			}
+		}
+
+		assertEquals(cities[0], "Ziguinchor");
```

Comment:
You could put the expected values into an array then loop through them, asserting each.